FirebaseStorage/Sources/FIRStorage.m

```diff
@@ -36,14 +36,59 @@
     NSMutableDictionary<NSString * /* bucket */, GTMSessionFetcherService *> *> *_fetcherServiceMap;
 static GTMSessionFetcherRetryBlock _retryWhenOffline;
 
-@interface FIRStorage () {
+//Should this be in FIRStorage_Private.h instead?
+@protocol FIRStorageInstanceProvider
+@end
+
+@interface FIRStorage () <FIRStorageInstanceProvider, FIRLibrary> {
   /// Stored Auth reference, if it exists. This needs to be stored for `copyWithZone:`.
   id<FIRAuthInterop> _Nullable _auth;
 }
 @end
 
 @implementation FIRStorage
 
++ (void)load {
+  // Register with Core as a library.  Adding FIRStorage as an Interop library enables
+  // proposed FirebaseExtended library SwiftLogForFireCloudStorage to depend on FIRStorage
+  // without making an explicit dependency of FIRStorage by SwiftLogForFireCloudStorage
+  [FIRApp registerInternalLibrary:(Class<FIRLibrary>)self
```

Comment:
Storage already registers as a library here: https://github.com/firebase/firebase-ios-sdk/blob/d261cdeb9bec45317c2d5086336e3a3b5f33e651/FirebaseStorage/Sources/FIRStorageComponent.m#L55

So this can be removed.